extensions/wikia/LookupContribs/SpecialLookupContribs_ajax.php

```diff
@@ -146,8 +135,8 @@ public static function prepareLookupUserData( $activityData, $username ) {
 				$row['id'], // wiki Id
 				$row['title'], // wiki title
 				$row['url'], // wiki url
-				$wg->Lang->timeanddate( wfTimestamp( TS_MW, $row['last_edit'] ), true ), // last edited
-				$wg->ContLang->formatNum( $row['editcount'] ),
+				$wg->Lang->timeanddate( wfTimestamp( TS_MW, $row['lastedit'] ), true ), // last edited
+				$wg->ContLang->formatNum( $row['edits'] ),
```

Comment:
Again, renamed so that this is the same value as what is used by the ajax requests